test/general/openpgp.js

```diff
@@ -928,6 +947,12 @@ export default () => describe('OpenPGP.js public api tests', function() {
     it('readPrivateKeys should throw on armored public keys', async function() {
       await expect(openpgp.readPrivateKeys({ armoredKeys: twoPublicKeys })).to.be.rejectedWith(/Armored text not of type private key/);
     });
+
+    it('readKey should work for a key that uses SHA3 internal signatures', async function() {
+      const key = await openpgp.readKey({ armoredKey: priv_key_sha3 });
+      expect(key.isPrivate()).to.be.true;
+      expect(key.isDecrypted()).to.be.true;
+    });
```

Comment:
```suggestion
```

No need to test this in isolation, because thanks to the forward compatibility changes included in v5.10.0, reading the key should work fine even without SHA3 support.
It's only when trying to use the key that errors would be triggered .
(At least, this should be true in principle, but in practice it's not happening due to a small oversight in the v6 signature implementation, which I'll fix shortly 😇 ).